tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -469,4 +469,49 @@ describe('People And Roles component for Change of Registration', () => {
     expect(wrapper.vm.$data.activeIndex).toBe(NaN)
     expect(wrapper.vm.$data.isAddingEditingOrgPerson).toBe(true)
   })
+
+  it('change button is not visible to users for SP where the sole proprietor is an organization', () => {
+    store.state.stateModel.tombstone.keycloakRoles = ['user']
+    store.state.stateModel.tombstone.entityType = 'SP'
+    store.state.stateModel.peopleAndRoles.orgPeople = [
+      {
+        officer: {
+          partyType: 'organization',
+          id: '0'
```

Comment:
You don't need the id for the getter, so prob better to leave it out. Also below.

Reply:
Nice, good catch